examples/quickstart/pom.xml

```diff
@@ -11,6 +11,8 @@
   </parent>
 
   <artifactId>reactive-messaging-quickstart</artifactId>
+  
+  <name>SmallRye Reactive Messaging : Quickstart :: Memory</name>    
```

Comment:
I think this should just be "Quickstart :: General" as though it's using in memory, I don't think that was the main purpose